src/classes/PReq_CannotDelete_TEST.cls

```diff
@@ -36,33 +36,111 @@
 */
 @isTest
 public with sharing class PReq_CannotDelete_TEST {
-
+  /*********************************************************************************************************
+    * @description Retrieves the Business Account record type Id. 
+    */
+    public static String bizAccRecordTypeId = UTIL_Describe_API.getBizAccRecTypeID(); 
```

Comment:
Given the underlying code in UTIL_Describe for this, I want to flag this for potential cleanup when we address that issue. The desired result here seems to be to get an account record type that does not have existing TDTM behavior being run against it.